views/pages/experimental/apply/form.njk

```diff
@@ -118,8 +118,8 @@
                             {% if field.isConditionalOn and fieldErrors | length < 1 %}data-conditional-field="{{ field.name }}" hidden{% endif %}>
                             <div class="form-field__main">
                                 <div class="form-field__body">
-                                    {% if field.type === 'text' %}
-                                        {{ inputText(field) }}
+                                    {% if field.type === 'text' or field.type === 'email' %}
```

Comment:
Starting to see how this could get awkward. My bad. Wonder if there's another way.